zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -1173,7 +1173,11 @@ public boolean deleteDataVisContent(Session session, Path contentPath) throws IO
             return false;
         }
 
-        String visualisationZipUri = contentPath.toString();
+        String visualisationZipUri = contentPath.getParent().toString();
+        if (visualisationZipUri == null || StringUtils.isEmpty(visualisationZipUri)){
+            info().data("zip", visualisationZipUri).log("unable to delete visualisation");
```

Comment:
Maybe add a bit more context to this log message - `"unable to delete visualisation path provided was null/empty"`.